compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/test/AMDAIEDmaUtilsTest.cpp

```diff
@@ -326,6 +332,9 @@ TEST_F(AccessPatternCombinationTest, CombineAccessPatterns) {
   checkCombineAccessPatterns({0, 0}, {16, 32}, {16, 1}, {1, 0, 0}, {3, 16, 32},
                              {64, 16, 1}, {0, 0, 0}, {4, 16, 32}, {64, 16, 1},
                              3);
+  // stride matching
```

Comment:
Could you add a few more test cases for this case with more dimensions and with non-zero offsets for A? E.g. something like this:
```
checkCombineAccessPatterns({4, 0}, {8, 16}, {64, 1}, {4 * 64 + 8 * 64, 0}, {1, 16},
                             {1, 1}, {4, 0}, {9, 16}, {64, 1}, 3, true);
```

Reply:
I've added this and a few others